rpm/qmf-qt5.spec

```diff
@@ -60,9 +60,8 @@ Patch26:  0026-Revert-Set-PLUGIN_CLASS_NAME-in-plugin-.pro-files.patch
 Patch27:  0027-Revert-Bump-version-to-6.0.0-since-we-build-against-.patch
 Patch28:  0028-Protect-service-pointer-access.patch
 Patch29:  0029-Save-account-before-config.patch
```

Comment:
Did we reach a conclusion that these two could be removed?

Reply:
Yes, I agree. I'll remove them in an additionnal commit while updating this PR with proper submodule pointer.